modules/sharedaddy/sharing-service.php

```diff
@@ -577,6 +577,10 @@ function sharing_maybe_enqueue_scripts() {
 }
 
 function sharing_add_footer() {
+	if ( Jetpack_AMP_Support::is_amp_request() ) {
+		return;
+	}
```

Comment:
👍 This will work fine because it will run at the `wp_footer` action, which happens after `wp`.